data/participants.json

```diff
@@ -110,7 +110,17 @@
             "github"   : "http://github.com/coderjolly",
             "twitter"  : "http://twitter.com/coderjolly",
             "referral" : ""
-        },
+        }
```

Comment:
You are missing a comma(,) here. This makes this json file invalid. You can test if your json code is valid or not, using https://jsonlint.com/

Reply:
thank U :)